spec/models/card_spec.rb

```diff
@@ -165,4 +165,11 @@
     card.check_translation("Словечко")
     expect(card.review_date).to eql(original_review_date)
   end
+
+  it "send notification about new card to review" do
+    user = create(:user)
+    user.cards.create(original_text: "Word",
+                     translated_text: "Слово", deck_id: 1)
+    expect{Card.mail_cards_to_review}.to change { ActionMailer::Base.deliveries.count }.by(1)
```

Comment:
Line is too long. [93/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.